docs/docs/behaviors/macros.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Macro Behavior
+sidebar_label: Macros
+---
+
+## Summary
+
+The macro behavior allows configuring a list of other behaviors to invoke
+when the macro is pressed and/or released.
+
+## Macro Definition
+
+Each macro you want to use in your keymap gets defined first, then bound in your keymap.
+
+A macro definition looks like:
+
+```
+/ {
+    macros {
+        zed_em_kay: zed_em_kay {
+            label = "ZM_zed_em_kay";
```

Comment:
Typo?

```suggestion
            label = "ZMK_zed_em_kay";
```

Reply:
No, was going for the initials for Zmk Macro -> "Z.M." -> `ZM_`

Reply:
Oh nice, I missed that 👍

Reply:
> Which is in fact what the macro generates.

Does [this example](https://github.com/zmkfirmware/zmk/pull/1166/files#diff-bb8c34fc4eb07da51cc48c0999057d683fed48c7c51997521ada620789efe88aR219) need changing at the end of the doc then? (`ZMK_my_macro`→`ZM_my_macro`)

```
    my_macro: my_macro {
        compatible = "zmk,behavior-macro";
        label = "ZMK_my_macro";
        #binding-cells = <0>;
        wait-ms = <30>;
        tap-ms = <40>;
        bindings = <&kp Z &kp M &kp K>;
    };
```

Reply:
Indeed. Fixed.